### DISCUSSION TITLE
how to define request body for telegram webhook handler?

### DISCUSSION THREAD

**Discussion by pirey** [fastapi/fastapi#7665] [Questions] [answered]
### First check

* [x] I used the GitHub search to find a similar issue and didn't find it.
* [x] I searched the FastAPI documentation, with the integrated search.
* [x] I already searched in Google "How to X in FastAPI" and didn't find any information.

### Description

I'm trying to write handler for telegram webhook.

Telegram will send `update` object whenever user sends message to my bot, and i need to process it inside my route handler.

The problem is, the telegram [update](https://core.telegram.org/bots/api#update) object is rather large, and has nested object of different types.

How can i easily define pydantic model to declare the `update` object schema?

Or, is there a better way to handle this?

**Comment by pirey** [comment, marked as answer]
i solve this issue by using type definition from pygramtic
https://github.com/devtud/pygramtic

**Comment by tiangolo** [comment]
Thanks for reporting back and closing the issue @pirey :+1:
